addons/website/static/src/scss/website.scss

```diff
@@ -872,10 +872,12 @@ header {
         }
     }
     // ... but we limit the navbar-collapse height
-    .navbar-collapse.show {
-        max-height: 80vh;
-        overflow-y: auto;
-        overflow-x: hidden; // Needed because of container in container having 0px padding... TODO improve
+    @include media-breakpoint-down(md) {
+        .navbar-collapse.show {
+            max-height: 80vh;
+            overflow-y: auto;
+            overflow-x: hidden; // Needed because of container in container having 0px padding... TODO improve
+        }
```

Comment:
I think that the the hamburger menu shouldn't have the `show` class when it's not visible. I would prefer a solution where you hide the hamburger menu content before it disappear. The actual solution would not be that robust if one day we have a hamburger menu displayed on lg screen but not on `xl` screen.
What do you think ?

Reply:
Replaced the implementation to remove `show` whenever the toggler becomes `display: none`. No more potential SCSS "surprises".